Doc/library/ipaddress.rst

```diff
@@ -219,9 +219,14 @@ write code that handles both IP versions correctly.  Address objects are
       ``"::abc:7:def"``.
    2. An integer that fits into 128 bits.
    3. An integer packed into a :class:`bytes` object of length 16, big-endian.
+   4. String IPv6 address may contain scope (or zone) id in format <address>%<scope_id>.
+      Scope id could not be blank or negative integer.
+      See :RFC:`4007` for details.
```

Comment:
Here's an alternate wording. One substantive change: negative integers are fine, and `%` is disallowed. (The RFC says non-negative integers *must* be supported, but everything else that's non-blank and has no `%` is OK to support.)

... Oh also, this should be part of point 1. Let's perhaps make it a second paragraph there.

... Oh hey, GitHub has this "suggest edit" feature. I'll try it.

Reply:
Sounds reasonable, fixed.